client/components/SingleProduct.js

```diff
@@ -64,21 +64,20 @@ class SingleProduct extends React.Component {
         <Grid>
```

Comment:
General comment for SingleProduct-- I made a lot of changes to this yesterday, so I think these will all conflict.  But it doesn't look like anything changed here really other than double-quotes becoming single-quotes.  

Is it possible to untrack this file from your branch in case it creates merge conflicts bringing mine in?